### ISSUE TITLE
Wrong format

### ISSUE THREAD

**Issue by 8nickB** [tokgozkerem/ucl_2024_draw_simulator#2]
In the Champions League format next year, there are no groups.
Each team could get different opponents, with the only rules being that teams from the same country aren‘t allowed to face each other and every team has to play against two teams from each drawing pot
Hope that helps

**Comment by snx888** [comment]
correct, I wanted to mention the same.... by searching for an algorithm to calculate the opponents I found this project, but unfortunately it also wasn't solved here ;)
my own tries always leads to less than 8 games per team, cause there was no possible opponent left :-/
